src/pages/index.md

```diff
@@ -1,4 +1,13 @@
 ---
+keywords:
+  - Creative Cloud
+  - API Documentation
+  - ExtendScript
+  - SDKDocumentation
```

Comment:
Missing space between SDK and Documentation

Reply:
Fixed